### DISCUSSION TITLE
how to bind keys in PlatfromerController2D ?

### DISCUSSION THREAD

**Discussion by builtinslib** [pokepetter/ursina#424] [Q&A]
I have 2 players in my code. I want to bind keys for player 2 but I cant do it. If I press WASD keys then 2 players moves together. I want to move first player with WASD and second player with arrow keys.

**Comment by pokepetter** [comment]
You can use input_handler.bind() to rebind keys:
```py
input_handler.bind('z', 'w')  # 'z'-key will now be registered as 'w'-key
```
